src/Paramore.Brighter.MessagingGateway.AWSSQS/RoutingKeyType.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Paramore.Brighter.MessagingGateway.AWSSQS;
+
+/// <summary>
+/// The routing key type
+/// </summary>
+public enum RoutingKeyType
```

Comment:
@iancooper do you think this is a good name or should I change to something else?

Reply:
Technically it is a ChannelType (according to Hohpe). The Routing Key is really the Topic, and whilst it is a different ARN, I would go with ChannelType